universe/Species.cpp

```diff
@@ -146,6 +146,25 @@ std::string Species::Dump() const {
     return retval;
 }
 
+std::string Species::GameplayDescription() const
+{
+    std::string result(UserString(m_gameplay_description));
+
+    for (std::vector<boost::shared_ptr<Effect::EffectsGroup> >::const_iterator it = m_effects.begin();
+         m_effects.end() != it; ++it)
+    {
+        if (it == m_effects.begin())
```

Comment:
I don't see how this should happen.

The condition is "is this the first element of the list?" and that has no relation with the description.

Empty strings are never displayed at all.